web/cobrands/fixmystreet/assets.js

```diff
@@ -116,6 +167,10 @@ function init_asset_layer(layer, pins_layer) {
         fixmystreet.map.events.register( 'zoomend', layer, check_zoom_message_visibility);
     }
 
+    if (layer.CLASS_NAME == 'OpenLayers.Layer.VectorAsset' || layer.CLASS_NAME == 'OpenLayers.Layer.VectorNearest') {
```

Comment:
This if will always be true, only layers of those two classes are created by this file.